### PR TITLE
Update microshift flow with secret and name update

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: .github/workflows/macos-microshift.yml [hunk 1]
```diff
@@ -39,11 +39,21 @@ jobs:
         run: make NO_CODESIGN=1 out/macos-universal/crc-macos-installer.pkg
       - name: Install crc pkg
         run: sudo installer -pkg out/macos-universal/crc-macos-installer.pkg -target /
+      - name: check if CRC_PULL_SECRET exists
+        env:
+          super_secret: ${{ secrets.CRC_PULL_SECRET }}
+        if: ${{ env.super_secret == '' }}
+        run: 'echo the secret \"CRC_PULL_SECRET\" has not been made; echo please go to \"settings \> secrets \> actions\" to create it'
```
Comment (cfergeau) [inline .github/workflows/macos-microshift.yml]: `made`? set?

File: .github/workflows/macos-microshift.yml [hunk 2]
```diff
@@ -13,6 +13,8 @@ on:
 jobs:
   build:
     runs-on: ${{ matrix.os }}
+    env:
+      SHELL: /bin/bash
```
Comment (cfergeau) [inline .github/workflows/macos-microshift.yml]: why do we need to use bash?
there's a 'standard' way of selecting the shell, `shell: bash` should work.

**Review by cfergeau — APPROVED** [review]

**Comment by praveenkumar** [comment]
/hold

Need to make sure microshift workflow is green. There is some issue wit workflow as it is not taking the pull secret from env.

File: .github/workflows/macos-microshift.yml [hunk 3]
```diff
@@ -35,6 +35,18 @@ jobs:
...
+      - name: check if CRC_PULL_SECRET exists
+        shell: bash
+        run: |
+          if [ "${{ secrets.CRC_PULL_SECRET }}" != '' ]; then
+             echo "the secret \"CRC_PULL_SECRET\" has not been set, please go to \"settings \> secrets \> actions\" to create it"
+             exit 1
+          fi
+      - name: Write the pull secret to json file
+        run: |
+          echo $CRC_PULL_SECRET > temp-ps.json
```
Comment (cfergeau) [inline .github/workflows/macos-microshift.yml]: Could this be ` echo ${{ secrets.CRC_PULL_SECRET }} > temp-ps.json` without the intermediate environment variable?

Comment (praveenkumar) [reply .github/workflows/macos-microshift.yml]: As per https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow it should be set as env or STDIN before using it from `run`

**Comment by praveenkumar** [comment]
Closing it from the fork and creating another PR from the branch.

**Pull request closed without merging**